idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -358,7 +365,10 @@ def run_chunk(self, part_suffix, input_files, chunk_count, lazy_run):
 
         index_dir_suffix = self.additional_attributes["index_dir_suffix"]
 
-        job_name = f"idseq-{deployment_environment}-{self.alignment_algorithm}"
+        pattern = r's3://idseq-samples-[a-z]+/samples/([0-9]+)/([0-9]+)/results'
```

Comment:
I tink we should make as few hard assumptions here as possible. Ideally these would be passed in as parameters, but if we must parse like this, I would restrict the regexp to `s3://.+/samples/([0-9]+)/([0-9]+)/`, and fall back to project_id, sample_id = 0, 0 when the regexp fails to match.